File: Problema8.py
# Pregunta por la edad y determina el precio del boleto
edad = int(input("Ingrese la edad: "))

if edad < 4:
    precio = 0
elif edad <= 12:
    precio = 5
elif edad <= 60:
    precio = 10
else:
    precio = 7

print(f"El precio de la entrada es: ${precio}")

File: Problema8.hs
ticketPrice :: Int -> Int
ticketPrice age
    | age < 4    = 0     
    | age <= 12  = 10    
    | age <= 60  = 20    
    | otherwise  = 12    

main :: IO ()
main = do
    putStrLn "Ingrese su edad:"
    input <- getLine
    let age = read input :: Int
    let price = ticketPrice age
    putStrLn $ "El precio del ticket es: " ++ show price ++ " COP."